backend/.env.example

```
@@ -210,6 +210,93 @@ WIKI_MAX_CONTENT_SIZE=10485760
+# Celery queue name for document conversion tasks
+# 文档转换任务的 Celery 队列名称
+KNOWLEDGE_CONVERSION_QUEUE=knowledge_conversion
+
+# Stale detection timeout for CONVERTING status (seconds)
+# 转换状态过期时间（秒），超过此时间视为卡住，可被重新入队
+KNOWLEDGE_INDEX_STALE_CONVERTING_SECONDS=1800
+
+# Conversion task distributed lock configuration
+# 转换任务分布式锁配置
+KNOWLEDGE_CONVERSION_LOCK_TIMEOUT_SECONDS=10000
+KNOWLEDGE_CONVERSION_LOCK_EXTEND_INTERVAL_SECONDS=60
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KNOWLEDGE_CONVERSION_LOCK_EXTEND_INTERVAL_SECONDS key should go before the KNOWLEDGE_CONVERSION_LOCK_TIMEOUT_SECONDS key

(UnorderedKey)